extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php

```diff
@@ -101,7 +99,7 @@ public function executePreview() {
 	private function getUploadErrorMessage( $status ) {
 		global $wgFileExtensions, $wgLang;
 
-		switch( $status[ 'status' ] ) {
+		switch ( $status['status'] ) {
 			case UploadBackgroundFromFile::FILESIZE_ERROR:
 				$msg = wfMsgHtml( 'themedesigner-size-error' );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)